recipes/isoquant/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+OUTDIR=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $OUTDIR
+mkdir -p $PREFIX/bin
+chmod a+x *.py
+cp -rp * "$OUTDIR/"
+cd $PREFIX/bin
+ln -s "../${OUTDIR#$PREFIX}"/*.py .
```

Comment:
You might restructure this package in the next release to work as a normal python module (i.e., make it installable with pip).